### PR TITLE
Changes for Skill Lab and better SDK integration

### PR DESCRIPTION
**Pull request by mattsanchez**
- Allows CLI configuration to be determined by environment variables (CORTEX_URI and CORTEX_TOKEN) for better integration into Skill Lab
- Sets apiEndpoint and token params in Action invoke requests if not provided.  Supports testing actions that use Cortex APIs.
- Sets activationId in Action invoke requests if not provided.  Supports testing of batch job actions.

### CODE REVIEW

File: src/commands/actions.js [hunk 1]
```diff
@@ -145,13 +146,22 @@ module.exports.InvokeActionCommand = class {
...
             params.properties['daemon.method'] = options.method;
         if (options.path)
             params.properties['daemon.path'] = options.path;
+
+        // Set the API Endpoint and Token if not specified
+        if (!params.apiEndpoint) params.apiEndpoint = profile.url;
+        if (!params.token) params.token = profile.token;
+
+        // Set the Activation ID if not specified
+        if (!params.activationId) params.activationId = uuid();
```
Comment (cdraper-CS) [inline src/commands/actions.js]: Dont believe this is necessary any more... backend will generate id if activationId not provided

Comment (mattsanchez) [reply src/commands/actions.js]: @cdraper-CS I found this not to be the case for Jobs v3.  Maybe this was fixed, but it caused the job to fail previously.

Comment (cdraper-CS) [reply src/commands/actions.js]: Yes, it definitely used to be a problem and was fixed

Comment (mattsanchez) [reply src/commands/actions.js]: I confirmed this is working now, removed the check.

**Pull request merged**